x-pack/plugins/lists/public/exceptions/api.ts

```diff
@@ -249,42 +249,46 @@ export const fetchExceptionListById = async ({
  * Fetch an ExceptionList's ExceptionItems by providing a ExceptionList list_id
  *
  * @param http Kibana http service
- * @param listId ExceptionList list_id (not ID)
- * @param namespaceType ExceptionList namespace_type
+ * @param listIds ExceptionList list_ids (not ID)
+ * @param namespaceTypes ExceptionList namespace_types
  * @param filterOptions optional - filter by field or tags
  * @param pagination optional
  * @param signal to cancel request
  *
  * @throws An error if response is not OK
  */
-export const fetchExceptionListItemsByListId = async ({
+export const fetchExceptionListsItemsByListIds = async ({
   http,
-  listId,
-  namespaceType,
-  filterOptions = {
-    filter: '',
-    tags: [],
-  },
+  listIds,
+  namespaceTypes,
+  filterOptions,
   pagination,
   signal,
 }: ApiCallByListIdProps): Promise<FoundExceptionListItemSchema> => {
-  const namespace =
-    namespaceType === 'agnostic' ? EXCEPTION_LIST_NAMESPACE_AGNOSTIC : EXCEPTION_LIST_NAMESPACE;
-  const filters = [
-    ...(filterOptions.filter.length
-      ? [`${namespace}.attributes.entries.field:${filterOptions.filter}*`]
-      : []),
-    ...(filterOptions.tags.length
-      ? filterOptions.tags.map((t) => `${namespace}.attributes.tags:${t}`)
-      : []),
-  ];
+  const filters: string = filterOptions
```

Comment:
What happens if `listIds.length !== namespaceTypes.length`? Maybe could use some extra validation around that...

Reply:
👍  The backend has validation for it `x-pack/plugins/lists/server/routes/find_exception_list_item_route.ts` (line 50)